com.github.bharatkalluri.gifup.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id" : "com.github.bharatkalluri.gifup",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.bharatkalluri.gifup",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
```

Comment:
For which reason the network is required?

Reply:
This is not required, my bad.